### ISSUE TITLE
Feature Request: Configuration option to include addons and folders

### ISSUE THREAD

**Issue by sam-ward** [thomasmauerer/hassio-addons#77]
Hi,

I've been caught as well by the 3 hrs timeout / "Invalid Numeric Literal" error as well.  In an effort to combat that i've excluded my MariaDB and Influx DBs from the back. And thats all good. But what I would like to do is explictly backup only the MariaDB for example. At the moment the only way to achieve that is to Exclude everything else, but the problem is if I then add another addon, it'll get caught in the backup as well until I remember to exlucde it. 

What would be good is if there was a  include_addons and include_folders configuration option, so that in my automation that triggers the MariaDB backup, I can include_addons: "mariadb" and then I dont have to worry about maintaining that logic any further.

Cheers

**Comment by RoadkillUK** [comment]
I agree, I need to backup both MariaDB and Influx and it would be much easier if I could 'include' rather than 'exclude'

Thanks :)

**Comment by richard-scott** [comment]
This can exclude/include folders and addons:
https://github.com/jcwillox/hass-auto-backup

**Comment by bcutter** [comment]
THE **most wanted** top 1 feature. Excluding introduces just so much downsides... I have several automations, scripts and the default addon timer. Once I install a. We addon I have to check every of those places to maintain the exclude list. That's really 90's style.

In theory from an amateurs point of view t should be so easy to copy and invert the existing logic and in the end offering a parameter for it.

Unfortunately this repo seems to only collect issues at the moment (at least for a year) not even speaking of new "features" as @thomasmauerer seems to be the only maintainer and dev being able to actually work on this great project.

What a pity 🙁
